addons/resource.language.en_gb/resources/strings.po

```diff
@@ -7931,45 +8146,45 @@ msgid "Delay channel switch"
 msgstr ""
 
 msgctxt "#19074"
-msgid "Active:"
+msgid "Enabled"
 msgstr ""
 
 msgctxt "#19075"
-msgid "Name:"
+msgid "Name"
 msgstr ""
 
 msgctxt "#19076"
-msgid "Folder:"
+msgid "Folder"
 msgstr ""
 
 #empty string with id 19077
 
 msgctxt "#19078"
-msgid "Channel:"
+msgid "Channel"
```

Comment:
I'd prefer not to do this in this PR, as it did not introduce any of the three(!) existing "Channel" strings (19078, 19148, 19029)